rootfs/etc/nginx/template/routes.tmpl

```diff
@@ -0,0 +1,28 @@
+{{ $backends := .Backends }}
+
+{
+{{ range $index, $upstream := $backends }}
+{{if $index}},{{end}}
+"{{ $upstream.Name }}" : {
+    "upstreamhashby" : "{{ $upstream.UpstreamHashBy }}",
+    "sessionaffinity" : {
+        "affinitytype" : "{{ $upstream.SessionAffinity.AffinityType }}",
+        "cookiesessionaffinity" : {
+            "name" : "{{ $upstream.SessionAffinity.CookieSessionAffinity.Name }}",
+            "hash" : "{{ $upstream.SessionAffinity.CookieSessionAffinity.Hash }}"
+        }
+    },
+    "endpoints" : [
+        {{ range $index, $endpoint := $upstream.Endpoints }}
+            {{if $index}},{{end}}
+            {
+                "hostname" : "{{ $endpoint.Address | formatIP }}",
+                "port" : {{ $endpoint.Port }},
+                "maxfails" : {{ $endpoint.MaxFails }},
+                "failtimeout" : {{ $endpoint.FailTimeout }}
+            }
+        {{ end }}
+    ]
+}
+{{ end }}
+}
```

Comment:
Maybe we can serialize the content of $backends in Go and avoid the template part? (with this are sure the JSON is valid)